src/main/java/io/vlingo/schemata/infra/persistence/SchemataObjectStore.java

```diff
@@ -0,0 +1,102 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+package io.vlingo.schemata.infra.persistence;
+
+import io.vlingo.actors.World;
+import io.vlingo.lattice.model.object.ObjectTypeRegistry;
+import io.vlingo.lattice.model.object.ObjectTypeRegistry.Info;
+import io.vlingo.schemata.infra.persistence.mappers.OrganizationStateMapper;
+import io.vlingo.schemata.model.Organization;
+import io.vlingo.schemata.model.OrganizationState;
+import io.vlingo.symbio.BaseEntry.TextEntry;
+import io.vlingo.symbio.State.TextState;
+import io.vlingo.symbio.store.DataFormat;
+import io.vlingo.symbio.store.common.jdbc.hsqldb.HSQLDBConfigurationProvider;
+import io.vlingo.symbio.store.dispatch.Dispatchable;
+import io.vlingo.symbio.store.dispatch.Dispatcher;
+import io.vlingo.symbio.store.object.MapQueryExpression;
+import io.vlingo.symbio.store.object.ObjectStore;
+import io.vlingo.symbio.store.object.PersistentObjectMapper;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiOnDatabase;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiOnHSQLDB;
+import io.vlingo.symbio.store.object.jdbc.jdbi.JdbiPersistMapper;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SchemataObjectStore {
```

Comment:
This class naming and API might be not the best but I wanted something quick to check if I'm on the good tracks.